test/e2e/storage/persistent_volumes.go

```diff
@@ -119,7 +119,7 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 
 	// Testing configurations of a single a PV/PVC pair, multiple evenly paired PVs/PVCs,
 	// and multiple unevenly paired PV/PVCs
-	ginkgo.Describe("NFS", func() {
+	ginkgo.Describe("NFS [Feature:StorageProviderGCEPD]", func() {
```

Comment:
NFS should not required PD afaik. Are you seeing failures?

Reply:
I thought in line 145 it created a PVC with empty storageClassName means using default storage class. Will be changing this